google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITProtoColumnTest.java

```diff
@@ -43,39 +34,104 @@
+    // Create a new testHelper with the cloud-devel host.
+    // testHelper = RemoteSpannerHelper.create(builder.build(),
+    // env.getTestHelper().getInstanceId());
+    RemoteSpannerHelper testHelper = env.getTestHelper();
+    databaseID = DatabaseId.of(testHelper.getInstanceId(), testHelper.getUniqueDatabaseId());
+    dbAdminClient = testHelper.getClient().getDatabaseAdminClient();
+    createDatabase();
+    databaseClient = testHelper.getClient().getDatabaseClient(databaseID);
+  }
+
+  public static void createDatabase() throws Exception {
+    final Database databaseToCreate =
+        dbAdminClient
+            .newDatabaseBuilder(databaseID)
+            .setProtoDescriptors("com/google/cloud/spanner/descriptors.pb")
+            .build();
+    final Database createdDatabase =
+        dbAdminClient
+            .createDatabase(
+                databaseToCreate,
+                Arrays.asList(
+                    "CREATE PROTO BUNDLE ("
+                        + "spanner.examples.music.SingerInfo,"
+                        + "spanner.examples.music.Genre,"
+                        + ")",
+                    "CREATE TABLE Singers ("
+                        + "  SingerId   INT64 NOT NULL,"
+                        + "  FirstName  STRING(1024),"
+                        + "  LastName   STRING(1024),"
+                        + "  SingerInfo spanner.examples.music.SingerInfo,"
+                        + "  SingerGenre spanner.examples.music.Genre,"
+                        + "  SingerNationality STRING(1024) AS (SingerInfo.nationality) STORED,"
+                        + "  ) PRIMARY KEY (SingerNationality, SingerGenre)",
+                    "CREATE TABLE Types ("
+                        + "  RowID INT64 NOT NULL,"
+                        + "  Int64a INT64,"
+                        + "  Bytes BYTES(MAX),"
+                        + "  Int64Array ARRAY<INT64>,"
+                        + "  BytesArray ARRAY<BYTES(MAX)>,"
+                        + "  ProtoMessage    spanner.examples.music.SingerInfo,"
+                        + "  ProtoEnum   spanner.examples.music.Genre,"
+                        + "  ProtoMessageArray   ARRAY<spanner.examples.music.SingerInfo>,"
+                        + "  ProtoEnumArray  ARRAY<spanner.examples.music.Genre>,"
+                        + "  ) PRIMARY KEY (RowID)",
+                    "CREATE INDEX SingerByNationalityAndGenre ON Singers(SingerNationality, SingerGenre)"
+                        + "  STORING (SingerId, FirstName, LastName)"))
+            .get(5, TimeUnit.MINUTES);
+
+    assertEquals(databaseID.getDatabase(), createdDatabase.getId().getDatabase());
+
+    // TODO(harsha): Check with backend team as this is not working for generated columns yet.
+    // GetDatabaseDdlResponse response =
+    // dbAdminClient.getDatabaseDdlResponse(databaseID.getInstanceId().getInstance(),
+    // databaseID.getDatabase());
+    // assertNotNull(response.getProtoDescriptors());
```

Comment:
We should make sure we remove these comments.

Reply:
This is needed to verify if the `proto_descriptor` field is populated while making a call to `getDatabaseDdlResponse` method. Currently the changes are not yet available and will uncomment it after that.